fork_choice_control/src/block_processor.rs

```diff
@@ -53,17 +121,24 @@ impl<P: Preset> BlockProcessor<P> {
                 )?;
 
                 let block_rewards = calculate_block_rewards(&slot_report);
+                info!("Block processed. Slot: {} | Rewards: {:?}", block.slot(), block_rewards);
+
 
                 Ok((state, Some(block_rewards)))
-            })
+            });
+        self.update_metrics("process_untrusted_block", start.elapsed());
```

Comment:
the name passed here is not identical to the method. In order to ensure this is always identical, you could use

```rust
        let function_name = tracing::Span::current().name().to_string();
        update_metrics(&function_name, duration, &self.metrics); 
```